src/main/scala/io/gatling/sbt/settings/gatling/TaskEnterpriseStart.scala

```diff
@@ -48,9 +48,7 @@ class TaskEnterpriseStart(config: Configuration, enterprisePackage: TaskEnterpri
     val enterprisePlugin = enterprisePluginTask(batchMode).value
 
     logger.info(s"Uploading and starting simulation...")
-    Using(enterprisePlugin) { enterprisePlugin =>
-      enterprisePlugin.uploadPackageAndStartSimulation(simulationId, systemProperties, environmentVariables, simulationClassname.orNull, file)
-    }.get
+    enterprisePlugin.uploadPackageAndStartSimulation(simulationId, systemProperties, environmentVariables, simulationClassname.orNull, file)
```

Comment:
Why don't you do a Try/recoverWith here too?

Reply:
There was a `.get` that threw the exception. (only enforce the close)
In this case, there is no point to have the Try/recoverWith